slowmovie.py

```diff
@@ -10,205 +10,233 @@
 # ** Waveshare library   **
 # *************************
 
-import os, time, sys, random 
-from PIL import Image
+import os, time, sys, random, signal
+from PIL import Image, ImageEnhance
 import ffmpeg
-import argparse
+from fractions import Fraction
 
-# Ensure this is the correct import for your particular screen 
-from waveshare_epd import epd7in5_V2
+try:
```

Comment:
Is it worth a try/except here? I think the configargparse library should either be a lib requirement or not. If we have good instructions on what modules to install it should be available. I understand the "fail gracefully" idea here since argparse shares the same method calls. I think it's confusing though to people who may try to use a conf file and it just doesn't work and they can't figure out why.

Reply:
I was originally trying not to add more dependencies, but I've been working out saving nowPlaying to the config file instead, which would make it a requirement anyway, so might as well.